tests/backstop_data/engine_scripts/puppet/civicase/manage.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+module.exports = async (engine, scenario, vp) => {
+  await require('./find.js')(engine, scenario, vp);
+  await engine.goto(await buildUrl.call(this, engine, 'a[title="Manage Case"]'), {'waitUntil': 'networkidle0'});
+  // open all accordions
+  await engine.evaluate(selector => {
+    document.querySelectorAll(selector).forEach(element => element.click());
```

Comment:
i believe you can do this in puppeteer. With `engine.$$()` you have the equivalent of `querySelectorAll()`, and from the resulting array you should be able to loop through it and call `engine.click()` for each element